sycl/test/basic_tests/image_api.cpp

```diff
@@ -0,0 +1,184 @@
+// RUN: %clang -std=c++11 -fsycl %s -o %t1.out -lstdc++ -lOpenCL -lsycl
+// RUNx: %clang -std=c++11 -fsycl %s -o %t2.out -lstdc++ -lOpenCL -lsycl -DSYCL_API
+// RUN: %clang -std=c++11 %s -o %t3.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUNx: env SYCL_DEVICE_TYPE=HOST %t2.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t3.out
+// RUN: %CPU_RUN_PLACEHOLDER %t1.out
+// RUNx: %CPU_RUN_PLACEHOLDER %t2.out
+// RUN: %GPU_RUN_PLACEHOLDER %t1.out
+// RUNx: %GPU_RUN_PLACEHOLDER %t2.out
+// RUN: %ACC_RUN_PLACEHOLDER %t1.out
+// RUNx: %ACC_RUN_PLACEHOLDER %t2.out
+
+// TODO Turn on t2.out test cases when SYCL image class api will be implemented
```

Comment:
I suggest remove them from this patch and add them with "image class api implementation."